### ISSUE TITLE
Update message once PRs have been merged or closed

### ISSUE THREAD

**Issue by danrabbit** [pullreminders/backlog#25]
I manage a very large team with over 100 repos and we have designated Tuesdays as "Reviews Day" when we go through and follow up on PRs that got swept under the rug that week.

When the bot posts a message with a large number of PRs, it would be nice if the message could be updated as PRs are closed so that we can easily see which PRs still need attention and which have already been delt with

**Comment by abinoda** [comment]
@danrabbit Just to be clear, you would like the Slack message to be edited as PRs are closed, right? I am not sure it is possible to do this... what do you think of the idea of having a UI in the pullreminders.com web app for this?

**Comment by danrabbit** [comment]
Yes, that's correct. The GitHub integration does a bit of this with checks on pull requests showing a colored bar next to each check and it updates when the status changes. Maybe that's a route to go?

**Comment by abinoda** [comment]
@danrabbit Thanks for that suggestion. I'll check that out.
